docs/src/main/asciidoc/security-authorization-of-web-endpoints-reference.adoc

```diff
@@ -4,7 +4,7 @@ and pull requests should be submitted there:
 https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
 ////
 [id="security-authorization-of-web-endpoints-reference"]
```

Comment:
@MichalMaler - Was just about to link to this great page from the security overview PR and realized I missed this suggestion:

Would it make sense to reduce length of filename to:

 `security-authorize-web-endpoints-reference.adoc` 

and the respective ID:

```suggestion
[id="security-authorize-web-endpoints-reference.adoc"]
```

Reply:
@michelle-purcell Yeah, why not :) Sure. Do you want me to do it in this PR or we can merge and you will do it in yours? What is better in your books?

Reply:
Renamed the file, changed the ID and fixed the xrefs :)

Reply:
Renaming patch here:
https://github.com/quarkusio/quarkusio.github.io/pull/1616

Reply:
@MichalMaler - It's your baby! ... Thanks. Looks great :+1: